recipe/meta.yaml

```diff
@@ -20,16 +20,17 @@ build:
 
 requirements:
   host:
-    - python >=3.6
+    - python
     - pip
     - setuptools
     - wheel
```

Comment:
The upstream source https://github.com/python/mypy/blob/c7b4714e1f5e3cb8f3fec7426b6538fe1a3dcab1/pyproject.toml#L6-L16 and conda-forge https://github.com/conda-forge/mypy-feedstock/blob/e0d9f90b3a924243fa31522e9e144aaa40797a93/recipe/meta.yaml#L55-L61 are using runtime and other dependencies in `host` too. I'm not sure if they are really needed in host

Reply:
no, they are not